docs/extensions/generate_bug_description.py

```diff
@@ -49,5 +53,39 @@ def generate_bug_description(language):
                     bug_description_page.write('\n')
                 bug_description_page.write('\n')
 
+    with codecs.open('generated/detectorListEnabled.inc', 'w', encoding='UTF-8') as enabled_detector_page:
+        with codecs.open('generated/detectorListDisabled.inc', 'w', encoding='UTF-8') as disabled_detector_page:
+            for element in findbugs.iterfind(".//Detector"):
+                hidden = parse_bool_attr(element, "hidden")
+                if hidden:
+                    continue
+                klass = element.get("class")
+                disabled = parse_bool_attr(element, "disabled")
+                speed = element.get("speed")
```

Comment:
The `speed` attribute is already deprecated, could you consider removing it from this document?

https://github.com/spotbugs/spotbugs/blob/51e586bed98393e53559a38c1f9bd15f54514efa/spotbugs/src/main/java/edu/umd/cs/findbugs/DetectorFactory.java#L125-L129

Reply:
Removed